[CI] SnapshotShutdownIT testRemoveNodeDuringSnapshot failing

**Build Scans:**
- [elasticsearch-intake #34876 / part1](https://gradle-enterprise.elastic.co/s/7ndqsnddhwta4)
- [elasticsearch-pull-request #122592 / part-1](https://gradle-enterprise.elastic.co/s/abuivoq6dlets)
- [elasticsearch-pull-request #122445 / part-1](https://gradle-enterprise.elastic.co/s/vnuhl42jvlpug)
- [elasticsearch-pull-request #122417 / part-1](https://gradle-enterprise.elastic.co/s/x2223jwwuebbe)
- [elasticsearch-periodic-platform-support #11890 / rocky-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/uhrb3okyv3wh4)
- [elasticsearch-intake #34834 / part1](https://gradle-enterprise.elastic.co/s/mtnyh5m3tpvck)
- [elasticsearch-pull-request #122157 / part-1](https://gradle-enterprise.elastic.co/s/w4aib43d72gbo)
- [elasticsearch-pull-request #122114 / part-1](https://gradle-enterprise.elastic.co/s/32hjrzbor65u2)
- [elasticsearch-pull-request #122050 / part-1](https://gradle-enterprise.elastic.co/s/5vv2qmzfxhlei)

**Reproduction Line:**
```
./gradlew ":server:internalClusterTest" --tests "org.elasticsearch.snapshots.SnapshotShutdownIT.testRemoveNodeDuringSnapshot" -Dtests.seed=6DFB118D1BD59622 -Dtests.locale=ks-Arab-IN -Dtests.timezone=America/Havana -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.snapshots.SnapshotShutdownIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testRemoveNodeDuringSnapshot'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: All incoming requests on node [node_s0] should have finished. Expected 0 bytes for requests in-flight but got 192 bytes; pending tasks [[{
  "node" : "node_s0",
  "id" : 1764,
  "type" : "transport",
  "action" : "internal:cluster/snapshot/update_snapshot_status",
  "description" : "",
  "start_time" : "2026-02-23T15:49:56.995Z",
  "start_time_in_millis" : 1771861796995,
  "running_time" : "4.4m",
  "running_time_in_nanos" : 264781045618,
  "cancellable" : false,
  "parent_task_id" : "iv97ALZmQ1yxWv7ENOq7Qw:39",
  "headers" : { }
}]]
```

**Issue Reasons:**
- [main] 9 failures in test testRemoveNodeDuringSnapshot (1.9% fail rate in 467 executions)
- [main] 2 failures in step part1 (3.1% fail rate in 64 executions)
- [main] 6 failures in step part-1 (2.2% fail rate in 274 executions)
- [main] 2 failures in pipeline elasticsearch-intake (3.1% fail rate in 64 executions)
- [main] 6 failures in pipeline elasticsearch-pull-request (2.2% fail rate in 270 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -431,18 +431,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
   method: test {csv-spec:spatial.ConvertFromStringParseError}
   issue: https://github.com/elastic/elasticsearch/issues/142843
-- class: org.elasticsearch.snapshots.SnapshotShutdownIT
-  method: testRemoveNodeDuringSnapshot
-  issue: https://github.com/elastic/elasticsearch/issues/142868
-- class: org.elasticsearch.snapshots.SnapshotShutdownIT
-  method: testAbortSnapshotWhileRemovingNode
-  issue: https://github.com/elastic/elasticsearch/issues/142869
-- class: org.elasticsearch.snapshots.SnapshotShutdownIT
-  method: testDeleteSnapshotWithPausedShardSnapshots
-  issue: https://github.com/elastic/elasticsearch/issues/142870
-- class: org.elasticsearch.snapshots.SnapshotShutdownIT
-  method: testStartRemoveNodeButDoNotComplete
-  issue: https://github.com/elastic/elasticsearch/issues/142871
 - class: org.elasticsearch.benchmark.vector.scorer.VectorScorerOSQBenchmarkTests
   method: testSingleScalarVsVectorized {p0=1024 p1=1 p2=NIO p3=DOT_PRODUCT}
   issue: https://github.com/elastic/elasticsearch/issues/142881

